desktop/desktop/lib/cookie-auth/index.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
```

Comment:
@mtias I'm looking at tackling the directory layout next and so wanted to ask your opinion on this first - do you mean all `desktop/desktop/lib` modules should be moved to `client/lib`, so we can get rid of the `desktop/desktop/lib` modules?